### ISSUE TITLE
setup script incompatible with gitea 1.20

### ISSUE THREAD

**Issue by dboreham** [cerc-io/hosting#45]
It said:

```
New user 'gitea_admin' has been successfully created!
This is your gitea access token: null. Keep it safe and secure, it can not be fetched again from gitea.
To use with laconic-so set this environment variable: export CERC_NPM_AUTH_TOKEN=null
jq: error (at <stdin>:1): Cannot index string with string "name"
Created the organization cerc-io
```

**Comment by dboreham** [comment]
Looks like the token issue API has changed:

```
$ curl -s http://localhost:3000/api/v1/users/gitea_admin/tokens -u gitea_admin:admin1234 -H 'Content-Type: application/json'
[]
```

**Comment by dboreham** [comment]
```
$ curl -s -X POST http://localhost:3000/api/v1/users/gitea_admin/tokens -u gitea_admin:admin1234 -H 'Content-Type: application/json' -d '{"name":"laconic-so-publication-token", "scopes": [ "sudo", "package" ] }'
{"message":"invalid access token scope provided: invalid access token scope: sudo","url":"http://gitea.local:3000/api/swagger"}
```

**Comment by zramsay** [comment]
good that they got rid of sudo as a scope, since it didn't mean what you'd assume 🤷

**Comment by dboreham** [comment]
Fix:

```
diff --git a/gitea/initialize-gitea.sh b/gitea/initialize-gitea.sh
index 5b34f64..9d4bf12 100755
--- a/gitea/initialize-gitea.sh
+++ b/gitea/initialize-gitea.sh
@@ -44,7 +44,7 @@ if [[ ${token_found} != 1 ]] ; then
     new_gitea_token=$( curl -s -X POST "${GITEA_URL_PREFIX}/api/v1/users/${GITEA_USER}/tokens" \
       -u ${GITEA_USER}:${GITEA_PASSWORD} \
       -H "Content-Type: application/json" \
-      -d '{"name":"'${CERC_GITEA_TOKEN_NAME}'", "scopes": [ "sudo", "package" ] }' \
+      -d '{"name":"'${CERC_GITEA_TOKEN_NAME}'", "scopes": [ "read:admin", "write:admin", "read:organization", "write:organization", "read:repository", "write:repository", "read:package", "write:package" ] }' \
       | jq -r .sha1 )
     echo "This is your gitea access token: ${new_gitea_token}. Keep it safe and secure, it can not be fetched again from gitea."
     echo "To use with laconic-so set this environment variable: export CERC_NPM_AUTH_TOKEN=${new_gitea_token}"
```

**Comment by dboreham** [comment]
I'll test the fix above against 1.19.3 and if it works, commit it. Otherwise it'll need to wait until 1.20 is out.

**Comment by dboreham** [comment]
This particular issue is fixed with PR #52
